packages/teleport-shared/src/utils/uidl-utils.ts

```diff
@@ -424,3 +424,16 @@ export const transformAttributesAssignmentsToJson = (
 
   return newStyleObject
 }
+
+export const traverseUIDLForRootElmNode = (node) => {
+  if (node && node.type) {
```

Comment:
why is this test needed?

Reply:
Not needed actually, I tried too many UIDL's and ended up doing this 😅 will remove 👍

Reply:
Yeah, I was doing this for the case of static and dynamic. I will throw an error from now on.